codeflash/cli_cmds/cmd_init.py

```diff
@@ -587,6 +587,16 @@ def configure_pyproject_toml(setup_info: SetupInfo) -> None:
         )
     elif formatter == "don't use a formatter":
         formatter_cmds.append("disabled")
+    if formatter in ["black", "ruff"]:
+        try:
+            result = subprocess.run([formatter], capture_output=True, check=False)
+            click.echo(f"✅ Formatter exists on system")
```

Comment:
we might not need to print this, we want to reduce the cognitive load for something thats not needed